### ISSUE TITLE
Cannot link to Google and problems with Public Address

### ISSUE THREAD

**Issue by AntonyBearpark** [d8ahazard/Phlex#282]
1. Running on Windows 10
2. New instance of XAMPP
3. Have you enabled the sockets module and ensured PHP has write-access to the directory containing Phlex? - No idea
4. Have you followed the (networking)[Talking-To-The-Outside-World] section? Yes
6. What are the last six digits of your server's API Token? 2e2573

Hello. I need some help. I either don't know what my Public address is or something is going terribly wrong. I've tried following all your guides and have even done some port forwarding but I cannot figure out how to connect phlex to my google account so it is ready for my google home setup. Every time I try to connect to google, I get: 

`Unable to connect to Client at http://###.###.#.##:80/Phlex, please check the setting for 'Public Address' in the Phlex UI.`

Please can anyone assist me on how to get this setup as I'm really struggling with it.

**Comment by leugenea** [comment]
I'm having same issue right now.

**Comment by fun4stuff** [comment]
Try changing your port to something other than 80.

**Comment by AntonyBearpark** [comment]
Tried to but still getting same problems.

**Comment by leugenea** [comment]
I've had issues with public address, my bad, sorry. Everything works fine now.

**Comment by AntonyBearpark** [comment]
None of these are helping me, I don't know what the problem is, yet others are fixing theirs without stating how?!

**Comment by jhawkey** [comment]
Hi @AntonyBearpark, I was having a similar issue. I'm pretty new to networking, and I think the github readme instructions leave out a few pieces of information that are useful to those that aren't super familiar with networking and ports etc. After some playing around, the following steps worked for me.

First, I went into Windows Firewall and set up a new port forwarding rule, following these instructions: http://www.tomshardware.com/faq/id-3114787/open-firewall-ports-windows.html
I just picked a random port number (in my case, 57226).

I then went to http://my.local.ip:57226/phlex to ensure I could still connect.

I then logged into my router, and added a port forwarding rule for my server. The internal and external port numbers were the same (57226).

Finally, in the phlex settings, under Public Address, I put in my public URL (found here https://whatismyipaddress.com/), and then :57226/phlex.

When I pressed the 'Test connectivity' button, it very quickly told me my connection was successful (with a little pop up box at the bottom of the screen). I was then able to link my accounts!

**Comment by AntonyBearpark** [comment]
Right, I have it connected, but just as it has connected to Google, my plex server has become unavailable. Complete one problem and hit another. Why isn't this easier?!

**Comment by jhawkey** [comment]
Unfortunately I don't have a solution to that issue! Perhaps someone else might know.
